src/gui/gtk.c

```diff
@@ -2667,6 +2667,32 @@ void dt_gui_load_theme(const char *theme)
     g_clear_error(&error);
   }
 
+  if(dt_conf_get_bool("themes/usertweaks"))
+  {
+    g_snprintf(usercsspath, sizeof(usercsspath), "%s/usertweaks.css", configdir);
+
+    if(g_file_test(path, G_FILE_TEST_EXISTS))
```

Comment:
should be `usercsspath` I imagine ;)

Reply:
Yes, though the code seems to work anyway!